handout/features/working_with_tsc.md

```diff
@@ -1,34 +1,34 @@
 # Working With `tsc`
 
 So far `tsc` has been used to compile a single file.  Typically developers have
-a _lot_ more than one file to compile.  `tsc` can handle multiple files as
+a _lot_ more than one file to compile. Thankfully  `tsc` can handle multiple files as
 arguments.
 
 Imagine two ultra simple files/modules:
 
 a.ts
-```js
+```ts
```

Comment:
Verified locally, thanks to [these updates to the highlight plugin](https://github.com/GitbookIO/plugin-highlight/compare/1.0.3...2.0.0) that Gitbook support made when I emailed them. Props to those guys.